bin/combine-pot-files.php

```diff
@@ -75,9 +75,90 @@ function woocommerce_admin_parse_pot( $file_name ) {
 	return $originals;
 }
 
+/**
+ * Generates a map with the mapping for 'original source file' -> final transpiled/minified file in the 'dist' folder.
+ * Format example:
+ * [
+ *   'client/card-readers/settings/file-upload.js' => [
+ *     'dist/index.js',
+ *     'dist/tos.js',
+ *   ]
+ * ]
+ *
+ * @return array Mapping of source js files and the generated files that use them.
+ */
+function load_js_transpiling_source_maps() {
```

Comment:
Can we make this stricter towards the types, please: `function load_js_transpiling_source_maps(): array`?